libs/remix-ui/checkbox/src/lib/remix-ui-checkbox.tsx

```diff
@@ -23,10 +25,12 @@ export const RemixUiCheckbox = ({
   checked,
   onChange,
   itemName,
-  categoryId
+  categoryId,
+  visibility,
+  display = 'flex'
 }: RemixUiCheckboxProps) => {
   return (
-    <div className="listenOnNetwork_2A0YE0 custom-control custom-checkbox" style={{ display: 'flex', alignItems: 'center' }} onClick={onClick}>
+    <div className="listenOnNetwork_2A0YE0 custom-control custom-checkbox" style={{ display: display, alignItems: 'center', visibility: visibility } as CSSProperties } onClick={onClick}>
```

Comment:
Is there a need to add `as CSSProperties`? It should compile fine without it.

Reply:
Yes, it shows TS issue without that.